### PR TITLE
added step-4: Conclusion in the step-by-step guide

### PR DESCRIPTION
**Pull request by Dishebh**
Changes made: 

- Added step-4: Optional: Conclusion in the `step-by-step guide` section of template-how-to.md

For issue: #58

### CODE REVIEW

File: how-to/template-how-to.md [hunk 1]
```diff
@@ -46,3 +46,7 @@ $ sudo apt install asciidoctor
 $ asciidoctor --version
 Asciidoctor 1.5.6.2 [https://asciidoctor.org]
 ```
+
+### Step 4: Optional: title for step - Conclusion
+
+Lead-in conclusion to the steps, explaining what happens after the user is done following the steps and what to expect next, maybe link to some related articles or the lead-in steps.
```
Comment (jaredmorgs) [inline how-to/template-how-to.md:52]: What do you think about this text instead:

>Provide a summary of the steps completed and explain what the user has achieved by following them. You can also include links to related articles that may help the reader reinforce concepts discussed in this how-to article.

Justification:

* This paragraph isn't a lead-in paragraph: it is a summary paragraph.
* The third comma-separated paragraph doesn't make sense in context of the others.

Comment (Loquacity) [reply how-to/template-how-to.md:52]: I like Jared's wording here.

Comment (adohertyTS) [reply how-to/template-how-to.md:52]: 👍  I also prefer Jared's wording.

Comment (Loquacity) [inline how-to/template-how-to.md:52]:
```suggestion
Provide a summary of the steps completed and explain what the user has achieved by following them. You can also include links to related articles that may help the reader reinforce concepts discussed in this how-to article.
```

Comment (Loquacity) [reply how-to/template-how-to.md:52]: Applied Jared's wording.

**Pull request merged**